config.env.py

```diff
@@ -20,8 +20,17 @@
 
 # Google OpenID Connect SSO config
 GOOGLE_ISSUER = env.get('GOOGLE_ISSUER', 'https://accounts.google.com')
-GOOGLE_CLIENT_ID = env.get('GOOGLE_CLIENT_ID', '')
-GOOGLE_CLIENT_SECRET = env.get('GOOGLE_CLIENT_SECRET', '------')
+GOOGLE_CLIENT_ID = env.get('GOOGLE_CLIENT_ID', '---')
+GOOGLE_CLIENT_SECRET = env.get('GOOGLE_CLIENT_SECRET', '---')
 
 LDAP_BIND_DN = env.get("LDAP_BIND_DN", default="cn=rides,ou=Apps,dc=csh,dc=rit,dc=edu")
 LDAP_BIND_PASS = env.get("LDAP_BIND_PASS", default=None)
+
+# Slack Config
+SLACK_TOKEN = env.get('SLACK_TOKEN','---')
+
+# Mail Config
+MAIL_SERVER = env.get("MAIL_SERVER", "thoth.csh.rit.edu")
+MAIL_USERNAME = env.get("MAIL_USERNAME", "rideboard@csh.rit.edu")
+MAIL_PASSWORD = env.get("MAIL_PASSWORD", None)
+MAIL_USE_TLS = True
```

Comment:
😢 missing newline

Reply:
?

Reply:
Github was showing there's no newline at the end of this file, which isn't a strict necessity, but the unix definition of a file involves a series of lines ending in new lines, and most command line tools are written with this as an expectation.